src/util/type_byte_size.cpp

```diff
@@ -28,33 +44,43 @@ BigInt member_offset_bits(const type2tc &type, const irep_idt &member)
     if(thetype.member_names[idx] == member.as_string())
       break;
 
-    result += type_byte_size_bits(it);
+    result += size_bits(it);
     idx++;
   }
 
   return result;
 }
 
-BigInt type_byte_size_default(const type2tc &type, const BigInt &defaultval)
+BigInt member_offset_bits(
+  const type2tc &type,
+  const irep_idt &member,
+  const namespacet *ns)
+{
+  return type_sizet(ns).member_offset_bits(type, member);
+}
+
+BigInt
+member_offset(const type2tc &type, const irep_idt &member, const namespacet *ns)
```

Comment:
const pointer -> const reference?

Reply:
If we're not going to start overloading all these functions (one with, one without namespace parameter), I need a way to say "optional". As the implementation would use a pointer anyway to represent this option (unless we go template-crazy), I thought this would be a cleaner API.